BUILD_WINDOWS_DEPS.md

```diff
@@ -0,0 +1,38 @@
+# Building and Uploading Windows C++20 Dependencies
+
+We have encountered an issue where the Windows build fails because it requires C++20 compatible binaries for dependencies (like Ruby), but only C++14 binaries are available on our Conan remote. To fix this, we need to manually build these dependencies with C++20 and upload them to the remote.
+
+A dedicated workflow file has been created for this purpose: `.github/workflows/build-windows-deps.yml`.
```

Comment:
Hum. Doubt it. And this is probably not the right place to do that.

I have jmarrec/conan-recipes (which should be hosted on the NREL umbrella, cf #5152)